.github/workflows/lint.yaml

```diff
@@ -0,0 +1,27 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    type: [created]
+    branches:
+      - '*'
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: install go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.16
```

Comment:
I just noticed the go version here is 1.14, any issues with that?
https://github.com/auth0/go-jwt-middleware/blob/575c421b5ffe5e97c21b47ef3e799490d581171f/go.mod#L3

Reply:
ah, good call - I'll bump this back to 1.14 for now

Reply:
@jfatta fixed!